Dockerfile.no_ML

```diff
@@ -0,0 +1,86 @@
+ARG PY_VERSION_DEFAULT=3.9
+
+FROM debian:bullseye as base
+
+USER root
+
+ARG TARGETARCH
+ARG PY_VERSION_DEFAULT
+ENV PY_VERSION ${PY_VERSION_DEFAULT}
+
+#essentials for production/dev
+RUN apt-get update && apt-get install -y \
+    libcurl4        `memgraph` \
+    libpython${PY_VERSION}   `memgraph` \
+    libssl-dev       `memgraph` \
+    openssl         `memgraph` \
+    build-essential `mage-memgraph` \
+    cmake           `mage-memgraph` \
+    curl            `mage-memgraph` \
+    g++             `mage-memgraph` \
+    python3         `mage-memgraph` \
+    python3-pip     `mage-memgraph` \
+    python3-setuptools     `mage-memgraph` \
+    python3-dev     `mage-memgraph` \
+    clang           `mage-memgraph` \
+    git             `mage-memgraph` \
+    unixodbc-dev        `mage-memgraph` \
+    --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY memgraph-${TARGETARCH}.deb .
+
+RUN dpkg -i memgraph-${TARGETARCH}.deb && rm memgraph-${TARGETARCH}.deb
```

Comment:
Dockerfile.release has a fix that was introduced in 1.12.1 release

`RUN dpkg -i memgraph-${TARGETARCH}.deb && rm memgraph-${TARGETARCH}.deb && rm /usr/lib/memgraph/query_modules/schema.so  \
&& rm /usr/lib/memgraph/query_modules/example_cpp.so`

Should this fix be applied here as well?

Reply:
So, we don't need that here, because these modules cause an issue only for pytorch modules, and here we don't use them.

Reply:
Thanks